addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_primitives.py

```diff
@@ -43,48 +44,69 @@ def gather_primitives(
     :return: a list of glTF2 primitives
     """
     primitives = []
+
+    blender_primitives = __gather_cache_primitives(blender_mesh_name,
+        vertex_groups, modifiers, export_settings)
+
+    for internal_primitive in blender_primitives:
+        material_idx = internal_primitive['material']
+        double_sided = False
+        material = None
+        if material_idx is not None:
+            blender_material = bpy.data.materials[material_names[material_idx]]
+            if bpy.app.version < (2, 80, 0):
+                double_sided = bpy.data.meshes[blender_mesh_name].show_double_sided
```

Comment:
So I have to ask, how far do we want to go to preserve the 2.79 per-mesh double-sided feature, given that the flag was dropped from 2.80?  Could we revert back to just using the custom node's double-sided setting in 2.79, which is a material setting, and that would save us some mesh-vs-material gymnastics that are really only here for the sake of 2.79?